pkgs/tools/admin/exoscale-cli/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, buildGo114Package, substituteAll, fetchgit }:
+
+let
+  version = "1.12.0";
+  rev = "4b82b898b56ea238eb80ac38da40dc8a460a7de7";
+in
+buildGo114Package {
```

Comment:
Why not `buildGoPackage`?

Reply:
The tool is developed and built with go 1.14 so I wanted to ensure this derivation is in line with that.

Reply:
Since you are the upstream maintainer as well, I think you can be trusted to update this to `buildGo115Package` in the future. But, it seems like an unnecessary maintenance burden to me, as the `buildGoPackage` function uses just the mainline Go compiler available, so if you trust Nixpkgs to always default `go` to the latest `go`, you should be fine with `buildGoPackage`.

Reply:
I want to make sure it's as close as possible to the binaries we publish.
This release was developed and tested with go 1.14 so I'd like to make sure the binary produced by nix does the same. In fact I just realized our readme is not up to date regarding this. :-)